.github/workflows/release.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
 
       - name: Trigger node-saucectl Release
-        if: github.event_name == 'release' && github.event.action == 'published'
+        if: ${{ !contains(github.ref, '-') }}
```

Comment:
Would be good to add a comment here, since it's not immediately obvious (later) why you'd be looking for a `-` character.

LGTM otherwise 👍